### PR TITLE
remove build noise from output

### PR DESCRIPTION
**Pull request by PeterPetrik**

### CODE REVIEW

File: src/quickgui/plugin/CMakeLists.txt [hunk 1]
```diff
@@ -97,14 +97,15 @@ IF(QMLPLUGINDUMP_FOUND)
...
+  # If the qmlplugindumpcommand fails, add -v argument to see detailed output for debugging
   SET(QGIS_QUICK_PLUGIN_TYPEINFO ${QGIS_QUICK_PLUGIN_RUNTIME_DIR}/qgsquick.qmltypes)
   ADD_CUSTOM_COMMAND(
       TARGET qgis_quick_plugin
       COMMAND ${CMAKE_COMMAND} -E make_directory ${QGIS_QUICK_TYPEINFO_GENERATE_DIR}
       COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_SOURCE_DIR}/qmldir ${QGIS_QUICK_TYPEINFO_GENERATE_DIR}
       COMMAND ${CMAKE_COMMAND} -E copy $<TARGET_FILE:qgis_quick_plugin> ${QGIS_QUICK_TYPEINFO_GENERATE_DIR}
       COMMAND ${QMLPLUGINDUMP_EXECUTABLE}
-      ARGS QgsQuick 0.1 . -v --output ${QGIS_QUICK_PLUGIN_TYPEINFO}
+      ARGS QgsQuick 0.1 . --output ${QGIS_QUICK_PLUGIN_TYPEINFO}
```
Comment (m-kuhn) [inline src/quickgui/plugin/CMakeLists.txt]: Should this version be put into a variable where it is easily discover- and manageable?

Comment (PeterPetrik) [reply src/quickgui/plugin/CMakeLists.txt]: This is for a verbose output of the qmlplugindump. Not very interesting at all, only if for some reason it is not working. What do you mean by put into variable?

Comment (m-kuhn) [reply src/quickgui/plugin/CMakeLists.txt]: Is QgsQuick *0.1* not a version string?

Comment (PeterPetrik) [reply src/quickgui/plugin/CMakeLists.txt]: ah, you are talking about that number... sure, that would be good to extract.

**Pull request merged**